server/bin/pbench-backup-tarballs.py

```diff
@@ -444,6 +461,10 @@ def main(cfg_name):
 
     logger = get_pbench_logger(_NAME_, config)
 
+    # We're going to need the Postgres DB to track dataset state, so setup
+    # DB access.
+    Database.init_db(config, logger)
+
```

Comment:
Isn't the database already initialized before in the shell.py? why do we need to do it here again?

Reply:
Two points:

1. This isn't part of the server, it's a standalone utility launched asynchronously by `crontab` ... it's possible that it may run before the server is started;
2. There's a bit of an ambiguity in "initializing" the database (creating the DB and tables) vs "connecting" to the database (setting up the engine and session to access it). Which means that even if we know the tables exist, the way the code is constructed now each independent process using our database still needs to call this.

If we break off the "database create" into a separate utility, the server shell and other commands only need to "connect", and that might be worth doing but isn't the most important thing to worry about now.

Reply:
Yeah probably sometime in the future we need to do that where a database create is a separate utility and every other piece just tries to connect to it